src/components/rocp_sdk/sdk_class.hpp

```diff
@@ -2,8 +2,10 @@
 #define __VENDOR_PROFILER_V1_H__
```

Comment:
this and `sdk_class.h` are still using the **same** component template include guard